src/docs/truffle/reference/truffle-commands.md

```diff
@@ -58,14 +58,15 @@ Options:
 Displays and sets user-level configuration options.
 
 ```shell
-truffle config [--enable-analytics|--disable-analytics] [[<get|set> <key>] [<value-for-set>]]
+truffle config [--enable-analytics|--disable-analytics] [[<get|set> <key>] [<list>] [<value-for-set>]]
 ```
 
 Options:
 
 * `--enable-analytics|--disable-analytics`: Enable or disable analytics.
 * `get`: Get a Truffle configuration option value.
 * `set`: Set a Truffle configuration option value.
+* `list`: List all Truffle configuration option values.
```

Comment:
Technically these are options but a better term would be sub-command, or config-command?

Reply:
What exactly is the change you are proposing?

Reply:
set `Options` to `sub-command`

Reply:
I'd be fine with that, at some point we will then have commands with both an `Options:` and `Sub-commands:` section. Good?